clang-tools-extra/clang-doc/support/File.cpp

```diff
@@ -0,0 +1,72 @@
+//===-- FileHelpersClangDoc.cpp - File Helpers -------------------*- C++-*-===//
```

Comment:
The filename in this comment should match the actual filename.

Reply:
Apparently, we don't use that line anymore. https://llvm.org/docs/CodingStandards.html#file-headers

Was brought up here when I saw this the first time: https://github.com/llvm/llvm-project/pull/111487#discussion_r2028191191